tests/foreman/longrun/test_oscap.py

```diff
@@ -386,20 +471,68 @@ def test_positive_oscap_run_via_ansible_bz_1814988(
 
     :BZ: 1814988
     """
+    contenthost = rex_contenthost
+    os_version = contenthost.os_version.major
+    distro = f'rhel{os_version}'
+    result = contenthost.register(
+        module_org,
+        None,
+        ak_name[distro],
+        target_sat,
+        ignore_subman_errors=True,
+        force=True,
+        insecure=True,
+    )
+    assert result.status == 0, f'Failed to register host: {result.stderr}'
+    rhel_repo = rhel_repos[distro]
+    profile = profiles[distro]
+    if distro not in ('rhel7'):
```

Comment:
The `()` here get removed. It's not a tuple so this is equivalent now:
```suggestion
    if distro not in 'rhel7':
```
Did you intend to check this?
```suggestion
    if distro not in ('rhel7',):
```